### DISCUSSION TITLE
get_iplayer error

### DISCUSSION THREAD

**Discussion by DrWho3000** [community/community#142970] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

is this a problem with get iplayer program or is it iplayer itself
ERROR: Failed to download BBC One schedule page (3/3): https://www.bbc.co.uk/schedules/p00fzl6n/2024/w43
ERROR: Response: 404 Not Found
.
ERROR: Failed to download BBC One schedule page (3/3): https://www.bbc.co.uk/schedules/p00fzl6n/2024/w44
ERROR: Response: 404 Not Found

**Comment by thomassiard** [comment]
It looks like the issue might be with the availability of the BBC schedule page rather than the get_iplayer program itself. The "404 Not Found" error means the requested schedule page URL doesn’t exist on the BBC website. This could happen if the schedule page URL format has changed, or the page for that specific week hasn’t been published yet.

You might want to:

Check if the URL structure has been updated by BBC, as sometimes these links change.
Try updating get_iplayer to the latest version, in case there are recent fixes for URL changes.
If possible, visit the BBC site manually to see if those specific pages are accessible.
If the pages load manually but get_iplayer still has trouble, it might be worth reporting the issue to the get_iplayer support team.

**Comment by ebndev** [comment]
Hi @DrWho3000,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on BBC website support. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
